python/cudf/cudf/_libxx/column.pyx

```diff
@@ -366,7 +371,7 @@ cdef class Column:
 
         cdef libcudf_types.bitmask_type* mask
         if self.nullable:
-            mask = <libcudf_types.bitmask_type*><uintptr_t>(self.base_mask_ptr)
+            mask = <libcudf_types.bitmask_type*><uintptr_t>(col.base_mask_ptr)
```

Comment:
`col` will not have a mask with it being `self.base_children[0]` so we need to handle this somehow